tests/execution-spec-tests/get_execution_spec_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# requires jq
+# sudo apt install jq
+
+# The following two artifacts are intended for consumption by clients:
+# - fixtures.tar.gz: Generated up to the last deployed fork.
+# - fixtures_develop.tar.gz: Generated up to and including the latest dev fork.
+# As of March 2024, dev is Prague, deployed is Cancun.
+
+ARTIFACT="fixtures_develop.tar.gz"
+TARGET_DIR="fixtures"
+
+OWNER="ethereum"
+REPO="execution-spec-tests"
+VERSION="v4.2.0" #"latest"
+DOWNLOAD_URL="https://github.com/$OWNER/$REPO/releases/download/$VERSION/$ARTIFACT"
+
+# Create target directory
+mkdir -p "$TARGET_DIR"
+
+# Download and extract
+echo "Downloading fixture artifacts $ARTIFACT version $VERSION..."
+curl -LO $DOWNLOAD_URL
+
+echo "Extracting artifacts to $TARGET_DIR..."
+tar -xzf "$ARTIFACT" --strip-components=1 -C "$TARGET_DIR"
+
+# Cleanup
+echo "Deleting tar file..."
+rm $ARTIFACT
```

Comment:
This seems like something we can have directly int he python testing harness instead of a separate script, and use the pytest fixtures instead of the makefile to manage downloading and extracting all the tests ahead of time.

Reply:
Yes, that was my initial approach. However, because each list of tests is computed when the module is loaded ([as, for example, here](https://github.com/runtimeverification/evm-semantics/blob/660cf3dff7bc8dc542d19ee58b18ce254b588798/kevm-pyk/src/tests/integration/test_execution_spec_tests.py#L29)), I couldn't have a fixture to download the tests before that.

Reply:
Ah OK. Thanks for teh explanation.